styx-scheduler-service/src/main/java/com/spotify/styx/state/handlers/ExecutionDescriptionHandler.java

```diff
@@ -72,8 +82,9 @@ public void transitionInto(RunState state, EventRouter eventRouter) {
     switch (state.state()) {
       case PREPARE:
         try {
+          final String executionId = createExecutionId();
```

Comment:
rename executionId to styxExecutionId ?

Reply:
I think `executionId` has been used basically all the places in code base. Changing it to `styxExecutionId` would not clear much of the confusing of the executionId vs flyteExecutionId.